manim/__main__.py

```diff
@@ -42,6 +42,9 @@ def open_file_if_needed(file_writer):
 
 
 def main():
+    console.log(
+        f"Manim Community Edition [green]v{__version__}[/green]", highlight=False
```

Comment:
I don't like `Edition` in here though.

Reply:
How about `The ManimCommunity's Manim`?
```suggestion
        f"The ManimCommunity's Manim [green]v{__version__}[/green]", highlight=False
```

Reply:
That's less conventional in English and is not consistent with our wording everywhere else.

Reply:
Well then how about 
```suggestion
        f"Manim Community [green]v{__version__}[/green]", highlight=False
```

Reply:
What's wrong with `Edition`? We use it all over the place

Reply:
We don't really, at least not in the library or the documentation: `grep -r edition .` doesn't return any results, and even searching for `manimce` doesn't return a whole lot of results (at least not since we have renamed the library from `manimce` to `manim`).

In writing, I usually distinguish between "3b1b's version" and "the community maintained version", I rarely use "edition" (but that is a personal preference of course).

That being said, I don't really care either way, and maybe just have a minor preference for "Manim Community v0.2.0" (but don't mind if the decision falls on pretty much any other version).

Reply:
I would say that because we do use the shortening CE, that we do use Edition. But, whatever everyone wants to use.

Reply:
Please use "Manim Community".